library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -413,7 +413,7 @@ interface ZoomApi {
      * @see zoom
      * @see realZoom
      */
-    fun setMaxZoom(maxZoom: Float, @ZoomType type: Int)
+    fun setMaxZoom(maxZoom: Float, @ZoomType type: Int = TYPE_ZOOM)
```

Comment:
Let's add JvmOverloads for Java users

Reply:
I'd love to do so, but this is currently not possible:
```
'@JvmOverloads' annotation cannot be used on interface methods
```
As a workaround I have added a method to the `ZoomApi` interface that has a default implementation.

Reply:
Oh I didn't know. So we can remove `= TYPE_ZOOM` then, right? I like the workaround signature because it has the `@Zoom` annotation.

Also could you do the same for min zoom? @markusressel

Reply:
Oh yeah totally forgot about minZoom, it's done.